CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
 # CHANGELOG
 
+## 2.0.0 (2022-11-14)
+
+MLflow 2.0.0 includes several breaking changes from the 1.x API, additional major features and improvements
+
+Breaking Changes:
```

Comment:
Can we also call out that support for Python 3.7 has been dropped & that MLflow 2.0 requires Python >= 3.8?

Reply:
Added